### PR TITLE
Tests for current user check requests

### PR DESCRIPTION
**Pull request by ChGen**
Tests for current user check requests. 1 new commit.

### CODE REVIEW

**Comment by ChGen** [comment]
@olenagerasimova , please review.

File: src/test/java/com/artipie/conan/http/UsersEntityTest.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ * @checkstyle ClassDataAbstractionCouplingCheck (999 lines)
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+public class UsersEntityTest {
+
+    @Test
+    public void userAuthTest() {
+        final Storage storage = new InMemoryStorage();
+        final String path = "/v1/users/authenticate";
+        final Response response = new UsersEntity.UserAuth(storage).response(
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/conan/http/UsersEntityTest.java:55]: these three variables are redundant and can be inlined in both tests methods

File: src/test/java/com/artipie/conan/http/UsersEntityTest.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            new RequestLine("GET", path, "HTTP/1.1").toString(),
+            Arrays.asList(
+                new MapEntry<>("Host", "localhost")
+            ), null
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/conan/http/UsersEntityTest.java:59]: Headers creation can be simplified to `new Headers.From("Host", "localhost")` in both tests

File: src/test/java/com/artipie/conan/http/UsersEntityTest.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+public class UsersEntityTest {
+
+    @Test
+    public void userAuthTest() {
+        final Storage storage = new InMemoryStorage();
+        final String path = "/v1/users/authenticate";
+        final Response response = new UsersEntity.UserAuth(storage).response(
+            new RequestLine("GET", path, "HTTP/1.1").toString(),
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/conan/http/UsersEntityTest.java:56]: `RequestLine` has more convenient ctor with two parameters: `RqMethod method` (enum) and `String uri`
```
new RequestLine(RqMethod.GET, "/v1/users/authenticate").toString()
```

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@ChGen thanks, please, check my suggestions

**Comment by ChGen** [comment]
@olenagerasimova , please review.

**Review by olenagerasimova — DISMISSED** [review]
@ChGen thanks

**Comment by ChGen** [comment]
@g4s8 , please review.

**Review by g4s8 — APPROVED** [review]

**Pull request merged**